### DISCUSSION TITLE
Anonymous problem

### DISCUSSION THREAD

**Discussion by ed-lam** [ds4dm/ml4co-competition#113] [Q&A]
Can you provide a MIP formulation of the anonymous problem?

**Comment by gasse** [comment]
Hi @ed-lam ,

The anonymous problem consists of a curated set of instances from the [MIRPLIB](https://mirplib.scl.gatech.edu/) library. We've updated the [data description](https://github.com/ds4dm/ml4co-competition/blob/main/DATA.md) page.

Best,
Maxime
